### DISCUSSION TITLE
Ψηφοφορία για συμπληρωματικές ΟΣΣ2

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#106] [Polls]
Για την καλύτερη υποστήριξη της μελέτης και της ΓΕ προτείνονται τακτικές συμπληρωματικές ΟΣΣ για τουλάχιστον δύο ΣΚ πριν την τυπική που ορίζει το ΕΑΠ

**Comment by iparthenios** [comment]
Καλημέρα σας!
Να υποθέσω ότι μιλάμε για τα Σαββατοκύριακα 19 και 26 Νοεμβρίου και όχι για την Κυριακή 3 Δεκεμβρίου;

**Reply by epidrome** [reply]
Για όλα αν αυτό βολεύει μια πλειοψηφία

Αν θέλετε μπορείτε να φτιάξετε και μια ξεχωριστή ψηφοφορία ειδικά για τις 3 Δεκ
